lib/rbac/filterer.rb

```diff
@@ -451,6 +448,11 @@ def method_with_scope(ar_scope, options)
     end
 
     def apply_scope(klass, scope)
+      klass = to_class(klass)
```

Comment:
This ends up calling to_class twice (once on line 177) and then again here.  Is that expected?